webapp/src/js/controllers/inbox.js

```diff
@@ -97,6 +97,12 @@ var feedback = require('../modules/feedback'),
       unknown: 'fa-question-circle',
     };
 
+    const updateReplicationStatus = status => {
+      $scope.replicationStatus.current = status;
```

Comment:
I suspect angular will re-paint this even if the values haven't changed which could hurt performance. Can we add a short-circuit check? eg:

```js
if ($scope.replicationStatus.current === status) {
  return;
}
```